motivaa-backend/motivaa-app/src/test/java/backendtesting/CookieHandler.java

```diff
@@ -0,0 +1,60 @@
+package backendtesting;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import lombok.extern.log4j.Log4j2;
+
+import java.util.Arrays;
+
+@Log4j2
+public class CookieHandler {
+    public static String retrieveJSessionIdCookie(String ENVIRONMENT_HOST,
+                                                  String endpoint,
+                                                  Boolean shouldRequestBeLogged) {
+
+        Response initialRequestResponse = initiateInitialRequest(
+                ENVIRONMENT_HOST,
+                endpoint,
+                shouldRequestBeLogged
+        );
+        return extractCookieFromResponseHeader(initialRequestResponse);
+    }
+
+    private static Response initiateInitialRequest(
+            String ENVIRONMENT_HOST,
+            String endpoint,
+            Boolean logEnabled) {
+        if (logEnabled) {
+            return RestAssured.given()
+                    .log().all()
```

Comment:
Also I am not sure what you think, but the indentation they use in the examples seem easier to read to me, like here in the examples:
https://github.com/rest-assured/rest-assured

Reply:
Super true. Done - thanks!